javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,86 @@
+let persons = [];
+let currentPerson;
+
+function getReply(command) {
+  command = command.toLowerCase();
+  const commandSplit = command.split(" ");
+  const pattern = /what is (\d+) ([+\-*/]) (\d+)/;
```

Comment:
Regexes can be very powerful, also pretty hard to debug.
Luckily, there are some [nice](https://regex-vis.com/) [tools](https://regex101.com/) out there.